etc/bash_completion.d/juju

```diff
@@ -256,39 +257,27 @@ print ("\n".join(
 }
 # List all controllers
 _JUJU_2_list_controllers_noflags() {
-    _JUJU_2_cache_cmd ${_JUJU_2_cache_TTL} cat \
```

Comment:
so when you say cache that exists you mean the local cache contents in .local/share/juju?

Reply:
right, the completion one is under `.cache/juju`, while from juju is under `.local/share/juju`